cmd/utils/flags.go

```diff
@@ -591,6 +592,57 @@ var (
 		Name:  "anchoring",
 		Usage: "Enable anchoring for service chain",
 	}
+	// ChainDataFetcher
+	EnableChainDataFetcherFlag = cli.BoolFlag{
+		Name:  "chaindatafetcher",
+		Usage: "Enable the ChainDataFetcher Service",
+	}
+	ChainDataFetcherNoDefault = cli.BoolFlag{
+		Name:  "chaindatafetcher.no.default",
+		Usage: "Turn off the starting of the chaindatafetcher",
+	}
```

Comment:
How about merging these two flags?

Reply:
Do you mean always auto-starting when `--chaindatafetcher` option is on?

Reply:
Right, when do we need a chain data fetcher which is given both EnableChainDataFetcherFlag and ChainDataFetcherNoDefault ?

Reply:
As discussed in f2f, it is helpful while debugging.